cmake/CMakeDependencies.cmake

```diff
@@ -10,6 +10,7 @@ set(CMAKE_PREFIX_PATH ${CMAKE_PREFIX_PATH} "${CMAKE_SOURCE_DIR}/win32-deps")
 # find JSONCPP
 # TODO: handle windows debug libraries!
 # TODO: fix FindJSONCPP file!
+hunter_add_package(jsoncpp)
 find_package(Jsoncpp)
```

Comment:
When using Hunter it should be `find_package(jsoncpp CONFIG REQUIRED)`:
* https://github.com/ruslo/hunter/wiki/pkg.jsoncpp